public/stylesheet.css

```
@@ -397,13 +396,23 @@ body.no-js .control-field__title {
 }
 
 .control__button {
+  border-bottom: 0.1rem solid var(--color-button-border);
+  border-top: 0.1rem solid var(--color-button-border);
+  border-right: 0.1rem solid var(--color-button-border);
```

This can be simplified to avoid repetition

```suggestion
  border: 0.1rem solid var(--color-button-border);
  border-left-width: 0;
```

With this change you can also have just `border-left: 0.1rem` in the `:first-of-type` ruleset.

---

ok